saddle-core/src/main/scala/org/saddle/Mat.scala

```diff
@@ -20,6 +20,7 @@ import scala.{specialized => spec}
 import java.io.OutputStream
 import org.saddle.index.{Slice, IndexIntRange}
 import org.saddle.mat.{MatImpl, MatMath}
+import scala.collection.compat.immutable.ArraySeq
```

Comment:
import scala.collection.immutable.ArraySeq
instead?
I don't think saddle cross-builds scala 2.12 and earlier.

Reply:
At this point it still does. I removed 2.12 in the scala3 pr.